gcloudutils/clients.go

```diff
@@ -0,0 +1,20 @@
+package gcloudutils
```

Comment:
One of the nice things about this client set in solobot is the ability to get clients based on the project_id, would be nice to somehow port that logic here as well.